js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -29,6 +29,50 @@ var join = require('path').join;
 var resolve = require('browser-resolve').sync;
 var emptyModule = join(__dirname, 'node_modules', 'browser-resolve', 'empty.js');
 
+// built in modules to shim
+// These are from browserify, and will need to be installed manually
```

Comment:
I assume these were from somewhere in the Browserify docs/ code, can you provide the link in case people want to see the source?